S3/Config.py

```diff
@@ -149,6 +149,8 @@ class Config(object):
     gpg_decrypt = u"%(gpg_command)s -d --verbose --no-use-agent --batch --yes --passphrase-fd %(passphrase_fd)s -o %(output_file)s %(input_file)s"
     use_https = True
     ca_certs_file = u""
+    ssl_key_file = u""
+    ssl_cert_file = u""
```

Comment:
I think that maybe it would be more easy for a "non expert" to understand these configs and cmd arguments if you rename them to:
> ssl_client_key_file
> ssl_client_cert_file